docs/gettingstarted/broadcast_data.md

```diff
@@ -33,8 +33,8 @@ nav_order: 4
 
 ## Additional info
 
-- Key Concepts: [Broadcast / shared data](/keyconcepts/broadcast.html)
-- Swagger: [POST /api/v1/namespaces/{ns}/broadcast/message](/swagger/swagger.html#/default/postBroadcastMessage)
+- Key Concepts: [Broadcast / shared data](/firefly/keyconcepts/broadcast.html)
```

Comment:
This broke the link for me on a local build of the docs, using the instructions here:
https://github.com/hyperledger/firefly/blob/main/docs/contributors/docs_setup.md#build-and-serve-the-docs-locally

However, I agree the main site is broken. So I think the issue here might be the local build instructions need to have a `/firefly` prefix somehow so that authors can test their changes correctly.

Reply:
I've raised https://github.com/hyperledger/firefly/pull/461 for that issue @nickgaski - hopefully that helps you with the review on this one.